packages/field-base/src/slot-target-mixin.js

```diff
@@ -38,28 +40,64 @@ const SlotTargetMixinImplementation = (superclass) =>
     }
 
     /**
-     * Forwards every node from the source slot to the target element
+     * Copies every node from the source slot to the target element
      * once the source slot' content is changed.
      *
      * @private
      */
-    __forwardNodesToSlotTarget() {
+    __checkAndCopyNodesToSlotTarget() {
+      this.__sourceSlotObserver.disconnect();
+
       if (!this._slotTarget) {
         return;
       }
 
+      // Remove any existing clones from the slot target
+      if (this.__slotTargetClones) {
+        this.__slotTargetClones.forEach((node) => {
+          if (node.parentElement === this._slotTarget) {
+            this._slotTarget.removeChild(node);
+          }
+        });
+        delete this.__slotTargetClones;
+      }
+
       // Exclude whitespace text nodes
       const nodes = this._sourceSlot
         .assignedNodes({ flatten: true })
         .filter((node) => !(node.nodeType == Node.TEXT_NODE && node.textContent.trim() === ''));
 
       if (nodes.length > 0) {
-        this._slotTarget.replaceChildren(...nodes);
+        this._slotTarget.innerHTML = '';
+        this.__copyNodesToSlotTarget(nodes);
       }
     }
+
+    /**
+     * Copies the nodes to the target element.
+     *
+     * @protected
+     * @param {Array<Node>} nodes
```

Comment:
If every node is expected not to be `null`, then the exclamation mark is probably required.

```suggestion
     * @param {!Array<!Node>} nodes
```